### ISSUE TITLE
Add accounting_group to Condor CDF dictionary

### ISSUE THREAD

**Issue by will-cern** [ganga-devs/ganga#352]
Hello,

My local condor batch system requires me to specify an 'accounting_group' in my configuration file. Please can you add this as an option to the Condor module.
This is pretty urgent, because without this I cannot use our batch system, so a hotfix would be much appreciated. I'm currently using 6.1.13

Cheers
Will

**Comment by egede** [comment]
@will-cern Please give some details on exactly how this should be specified and how the exact syntax is if used from the command line. While not familiar with how Condor works, I wonder if you could solve your problem using the `submit_options` property of the `Condor` backend.

**Comment by will-cern** [comment]
Hi,

I had a look at the submit_options too but this isn't something that goes in the command line, it's something that goes in the configuration file. Take a look at the "universe" property in the Condor module, for example. It needs to appear in the generated job in a similar way.

It's just:

accounting_group = 'string'

**Comment by will-cern** [comment]
My bad, I mean:

accounting_group = string

No quotes

**Comment by drmarkwslater** [comment]
@will-cern I'll try to get this in ASAP but until then, you should be able to get around it with a bit of minor hacking. If you specify:
```
j.backend.requirements.other = ['Memory >= 0\naccounting_group = string']
```
This will get added to the submit file correctly.

**Comment by will-cern** [comment]
Ok I'll give that a try. I thought the requirements got 'and'ed together, but if this 'other' property is special then yes it might work

	Cheers
		Will

**Comment by drmarkwslater** [comment]
They do, but if you put in the `\n` and make sure it's the last entry, it can get round it, SQL-injection style :smile:

**Comment by will-cern** [comment]
Love it!

While I'm here, could I also ask for that feature of --inPfnListFile option of ganga athena to be modified so that lines beginning with "#" in the input file area ignored. That would be very useful too

Cheers
	Will

**Comment by drmarkwslater** [comment]
You certainly could but could you create a new issue for it? Otherwise 
I'll forget :)

Thanks,

Mark

**Comment by will-cern** [comment]
Hi,

Can I also ask for the stream_output and stream_error options in the job description file to be made configurable too. Currently they both default to False, but it would be nice to be able to try switching them to True so that I can monitor the progress of a subjob.

Thanks
Will

**Comment by will-cern** [comment]
Hi.

Can I check on the progress of this and my other open issues? Just want to get an idea of when 6.1.19 will be available so I can try them all out

Thanks
Will

**Comment by drmarkwslater** [comment]
@will-cern We've slowed down development in the last week because the CERN VMs we use for testing seem to be acting up and it's making it difficult to know if we've broken anything :smile: We're going to shift to a different machine by the end of the week that should make life a lot easier. In addition, pretty much all the devs are are at a GridPP meeting until Thursday! However, I have every intention of getting this stuff in for 6.1.19 which will go out mid-next week (testing permitting!).

**Comment by will-cern** [comment]
Thanks for the update.

Is there any way I can help? Do we have a way for me to 'setup the trunk version' of ganga, so I can effectively do the testing for you, at least for this functionality

	Cheers
		Will

**Comment by drmarkwslater** [comment]
Hi Will,

To make sure you get all the externals you need, probably the easiest thing to do is to use the ganga-install script:

http://ganga.web.cern.ch/ganga/download/ganga-install

run this with:

ganga-install --extern=GangaAtlas,GangaPanda 6.1.18

then download the `develop` branch (or whatever branch you're interested in) and unzip it into:

`~/Ganga/install/.`

Setup Ganga as you normally would but then run it with:

`~/Ganga/install/<branch_name>/bin/ganga`

I have now very nearly fixed everything I was working on with Core (just need to get the tutorials done now!) and then I'm going to get to this and some other quick issues to be done for 6.1.19. If you keep an eye on the github page you'll see the PRs going in on Monday hopefully!

Thanks,

Mark

**Comment by drmarkwslater** [comment]
This should be fixed with #387
